File: elementary_computer_science/Pascal/divisible_by.pas
program divisible_by;
var a, b : integer;
begin
	write('Enter a = ');
	readln(a);
	write('Enter b = ');
	readln(b);
	if a mod b = 0 then writeln(a,' is divisible by ', b, '.') else writeln(a,' is not divisible by ', b, '.');
end.

File: elementary_computer_science/C/divisible_by.c
#include <stdio.h>
int main() {
	int a, b;
	scanf("%d", &a);
	scanf("%d", &b);
	if (a % b == 0) printf("%d is divisible by %d.\n", a, b);
	else printf("%d is not divisible by %d.\n", a, b);
	return 0;
}
// Terminal: g++ -std=c++11 -O2 -Wall divisible_by.c -o divisible_by